python/examples/mote-api.py

```diff
@@ -51,13 +64,134 @@ def mote_on(status):
     mote.show()
     return True
 
+
+## One Light, on One Mote Stick, Travelling at the set colour & channel from the USB port to the End
+def larson_usb_to_end(colour,channel):
+    r, g, b = hex_to_rgb(colour)
+    for i in range(0,16,1):
+        mote.set_pixel(channel,i,r,g,b)
+        mote.show()
+        time.sleep(pause_time_default)
+        mote.clear()
+
+## One Light, on One Mote Stick, Travelling at the set colour & channel from the End to the USB port
+def larson_end_to_usb(colour,channel):
+    r, g, b = hex_to_rgb(colour)
+    for i in range(15,-1,-1):
+        mote.set_pixel(channel,i,r,g,b)
+        mote.show()
+        time.sleep(pause_time_default)
+        mote.clear()
+
+## One Light, on Two Mote sticks at a time, Travelling at the set colour & channels from the USB port to the End
+def larson_double_usb_to_end(colour,ch1,ch2):
+    r, g, b = hex_to_rgb(colour)
+    for i in range(0,16,1):
+        mote.set_pixel(ch1,i,r,g,b)
+        mote.set_pixel(ch2,i,r,g,b)
+        mote.show()
+        time.sleep(pause_time_default)
+        mote.clear()
+
+## One Light, on Two Mote sticks at a time, Travelling at the set colour & channels from the End side to the USB port
+def larson_double_end_to_usb(colour,ch1,ch2):
+    r, g, b = hex_to_rgb(colour)
+    for i in range(15,-1,-1):
+        mote.set_pixel(ch1,i,r,g,b)
+        mote.set_pixel(ch2,i,r,g,b)
+        mote.show()
+        time.sleep(pause_time_default)
+        mote.clear()
+
+## One Light, on Four Mote sticks at a time, Travelling at the set colour & channels from the USB port to the End
+def larson_quad_usb_to_end(colour):
+    r, g, b = hex_to_rgb(colour)
+    for i in range(0,16,1):
+        mote.set_pixel(1,i,r,g,b)
+        mote.set_pixel(2,i,r,g,b)
+        mote.set_pixel(3,i,r,g,b)
+        mote.set_pixel(4,i,r,g,b)
+        mote.show()
+        time.sleep(pause_time_default)
+        mote.clear()
+
+## One Light, on Four Mote sticks at a time, Travelling at the set colour & channels from the End side to the USB port
+def larson_quad_end_to_usb(colour):
+    r, g, b = hex_to_rgb(colour)
+    for i in range(15,-1,-1):
+        mote.set_pixel(1,i,r,g,b)
+        mote.set_pixel(2,i,r,g,b)
+        mote.set_pixel(3,i,r,g,b)
+        mote.set_pixel(4,i,r,g,b)
+        mote.show()
+        time.sleep(pause_time_default)
+        mote.clear()
+
+
+## One Light, remaining On, Travelling at the set colour & channel from the USB port to the End
+def colour_flow_usb_to_end(colour,channel):
+    r, g, b = hex_to_rgb(colour)
+    for i in range(0,16,1):
+        mote.set_pixel(channel,i,r,g,b)
+        mote.show()
+        time.sleep(pause_time_default)
+
+## One Light, remaining On, Travelling at the set colour & channel from the End to the USB port
+def colour_flow_end_to_usb(colour,channel):
+    r, g, b = hex_to_rgb(colour)
+    for i in range(15,-1,-1):
+        mote.set_pixel(channel,i,r,g,b)
+        mote.show()
+        time.sleep(pause_time_default)
```

Comment:
Those threading changes might be a fair bit out of my current skill range.

Right now I'm using the pauses as part of the animation effects, so some type of wait has to be there.

I'll get up to speed on making a MoteEffect class.